Health check logging: ElapsedMilliseconds in messages makes logs harder to process

Disclamer: I think logging the duration of health checks is great.

I also think it should be logged as debug only. The current code logs the duration in warnings and errors too - those who are important to monitor in production. In Splunk we group log messages before creating a daily report. When duration is the only differing factor our reports look like this:

![image](https://user-images.githubusercontent.com/4621581/53100467-1fccf300-3528-11e9-89d6-5e6593eccc5c.png)

---
__Proposal__: only include ElapsedMilliseconds in debug messages

And the link to the code

https://github.com/aspnet/Extensions/blob/c3740ed31f55d22c39d49d17b449eb49b8fdffba/src/HealthChecks/HealthChecks/src/DefaultHealthCheckService.cs#L158

Hi @pranavkm , I've cloned the repo and followed build guide available [here](https://github.com/dotnet/aspnetcore/blob/master/docs/BuildFromSource.md). I'm getting errors while running the build command. Could you guide me?

```Building of C# project is enabled and has dependencies on NodeJS projects. Building of NodeJS projects is enabled since node is detected in C:\Program Files.
Detected JDK in C:\Program Files\Java\jdk-13.0.1 (via PATH)

  Determining projects to restore...
  Tool 'dotnet-serve' (version '1.7.125') was restored. Available commands: dotnet-serve
  
  Restore was successful.
  All projects are up-to-date for restore.
dotnet-install: .NET Core Runtime version 2.1.18 is already installed.
dotnet-install: Adding to current process PATH: "D:\Workspace\aspnetcore\.dotnet\". Note: This change will not be visible if PowerShell was run as a child process.
dotnet-install: .NET Core Runtime version 6.0.0-alpha.1.20428.2 is already installed.
dotnet-install: Adding to current process PATH: "D:\Workspace\aspnetcore\.dotnet\". Note: This change will not be visible if PowerShell was run as a child process.
dotnet-install: .NET Core Runtime version 6.0.0-alpha.1.20428.2 is already installed.
dotnet-install: Adding to current process PATH: "D:\Workspace\aspnetcore\.dotnet\x86\". Note: This change will not be visible if PowerShell was run as a child process.
dotnet-install: ASP.NET Core Runtime version 3.1.4 is already installed.
dotnet-install: Adding to current process PATH: "D:\Workspace\aspnetcore\.dotnet\". Note: This change will not be visible if PowerShell was run as a child process.
  Determining projects to restore...
  All projects are up-to-date for restore.
  RepoTasks -> D:\Workspace\aspnetcore\artifacts\bin\RepoTasks\Release\net5.0\RepoTasks.dll
  RepoTasks -> D:\Workspace\aspnetcore\artifacts\bin\RepoTasks\Release\net472\RepoTasks.dll

  Tool 'dotnet-serve' (version '1.7.125') was restored. Available commands: dotnet-serve
  
  Restore was successful.
  Restore completed in 57.66 ms for C:\Users\hanna\.nuget\packages\microsoft.dotnet.arcade.sdk\5.0.0-beta.20431.1\tools\Tools.proj.
dotnet-install: .NET Core Runtime version 2.1.18 is already installed.
dotnet-install: Adding to current process PATH: "D:\Workspace\aspnetcore\.dotnet\". Note: This change will not be visible if PowerShell was run as a child process.
dotnet-install: .NET Core Runtime version 6.0.0-alpha.1.20428.2 is already installed.
dotnet-install: Adding to current process PATH: "D:\Workspace\aspnetcore\.dotnet\". Note: This change will not be visible if PowerShell was run as a child process.
dotnet-install: .NET Core Runtime version 6.0.0-alpha.1.20428.2 is already installed.
dotnet-install: Adding to current process PATH: "D:\Workspace\aspnetcore\.dotnet\x86\". Note: This change will not be visible if PowerShell was run as a child process.
dotnet-install: ASP.NET Core Runtime version 3.1.4 is already installed.
dotnet-install: Adding to current process PATH: "D:\Workspace\aspnetcore\.dotnet\". Note: This change will not be visible if PowerShell was run as a child process.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj(31,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj(31,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\CommonLibTests\CommonLibTests.vcxproj(33,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\CommonLib\CommonLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\gtest\gtest.vcxproj(31,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\IISLib\IISLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj(30,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj(30,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\RequestHandlerLib\RequestHandlerLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\IISLib\IISLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\CommonLib\CommonLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\CommonLibTests\CommonLibTests.vcxproj(33,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\gtest\gtest.vcxproj(31,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj(30,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj(30,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\RequestHandlerLib\RequestHandlerLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.

Build FAILED.

D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj(31,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj(31,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\CommonLibTests\CommonLibTests.vcxproj(33,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\CommonLib\CommonLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\gtest\gtest.vcxproj(31,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\IISLib\IISLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj(30,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj(30,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\RequestHandlerLib\RequestHandlerLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\IISLib\IISLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\CommonLib\CommonLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\CommonLibTests\CommonLibTests.vcxproj(33,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\gtest\gtest.vcxproj(31,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj(30,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj(30,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
D:\Workspace\aspnetcore\src\Servers\IIS\AspNetCoreModuleV2\RequestHandlerLib\RequestHandlerLib.vcxproj(28,3): error MSB4019: The imported project "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Confirm that the expression in the Import declaration "D:\Workspace\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk.
    0 Warning(s)
    16 Error(s)

Time Elapsed 00:00:20.67
Build failed.
```

--- a/src/HealthChecks/HealthChecks/src/DefaultHealthCheckService.cs
+++ b/src/HealthChecks/HealthChecks/src/DefaultHealthCheckService.cs
@@ -191,7 +191,7 @@ namespace Microsoft.Extensions.Diagnostics.HealthChecks
             private static readonly Action<ILogger, double, HealthStatus, Exception?> _healthCheckProcessingEnd = LoggerMessage.Define<double, HealthStatus>(
                 LogLevel.Debug,
                 EventIds.HealthCheckProcessingEnd,
-                "Health check processing completed after {ElapsedMilliseconds}ms with combined status {HealthStatus}");
+                "Health check processing with combined status {HealthStatus} completed after {ElapsedMilliseconds}ms");
 
             private static readonly Action<ILogger, string, Exception?> _healthCheckBegin = LoggerMessage.Define<string>(
                 LogLevel.Debug,
@@ -199,7 +199,7 @@ namespace Microsoft.Extensions.Diagnostics.HealthChecks
                 "Running health check {HealthCheckName}");
 
             // These are separate so they can have different log levels
-            private static readonly string HealthCheckEndText = "Health check {HealthCheckName} completed after {ElapsedMilliseconds}ms with status {HealthStatus} and description '{HealthCheckDescription}'";
+            private static readonly string HealthCheckEndText = "Health check {HealthCheckName} with status {HealthStatus} completed after {ElapsedMilliseconds}ms with message '{HealthCheckDescription}'";
 
             private static readonly Action<ILogger, string, double, HealthStatus, string?, Exception?> _healthCheckEndHealthy = LoggerMessage.Define<string, double, HealthStatus, string?>(
                 LogLevel.Debug,

